spec/fixtures/api/no-leak.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Document</title>
+</head>
+<body>
+  <script>
+  window.postMessage({
+    require: typeof require,
+    exports: typeof exports,
+    windowRequire: typeof window.require,
```

Comment:
it's unclear to me in what circumstances `window.require` would refer to a different object than `require` here; what's this testing for? (same for `window.exports`)

Reply:
Unclear to me as well, but hey, better to test implict and explicit globals rather than let it break when something else funky happens

Reply:
I disagree; don't test something unless you actually understand what you're testing for :)

At the very least, be consistent with `sandboxed_preload`.